Framework/PythonInterface/plugins/algorithms/ConvertWANDSCDtoQ.py

```diff
@@ -309,29 +415,70 @@ def PyExec(self):  # noqa C901
 
         s1offset = np.deg2rad(self.getProperty("S1Offset").value)
         s1offset = np.array([[np.cos(s1offset), 0, np.sin(s1offset)], [0, 1, 0], [-np.sin(s1offset), 0, np.cos(s1offset)]])
+        for sym_op in sym_ops:
+            _output = np.zeros_like(output)
+            _outputr = _output.ravel()
+
+            _output_scale = np.zeros_like(output)
+            _output_scaler = _output_scale.ravel()
 
-        for n in range(number_of_runs):
-            R = inWS.getExperimentInfo(0).run().getGoniometer(n).getR()
-            R = np.dot(s1offset, R)
-            RUBW = np.dot(R, UBW)
-            q = np.round(np.dot(np.linalg.inv(RUBW), qlab.T) / bin_size - offset).astype(int)
-            q_index = np.ravel_multi_index(q, (dim0_bins + 2, dim1_bins + 2, dim2_bins + 2), mode="clip")
-            q_uniq, inverse = np.unique(q_index, return_inverse=True)
-            outputr[q_uniq] += np.bincount(inverse, data_array[:, :, n].ravel("F"))
-            output_scaler[q_uniq] += np.bincount(inverse) * scale[n]
             if _norm:
-                output_normr[q_uniq] += np.bincount(inverse, norm_array.ravel("F"))
-                output_norm_scaler[q_uniq] += np.bincount(inverse)
+                _output_norm = np.zeros_like(output)
+                _output_normr = _output_norm.ravel()
+                _output_norm_scale = np.zeros_like(output)
+                _output_norm_scaler = _output_norm_scale.ravel()
+            if _bkg:
+                _output_bkg = np.zeros_like(output)
+                _output_bkgr = _output_bkg.ravel()
+                _output_bkg_scale = np.zeros_like(output)
+                _output_bkg_scaler = _output_bkg_scale.ravel()
+
+            S = np.zeros((3, 3))
+            S[:, 0] = sym_op.transformHKL([1, 0, 0])
+            S[:, 1] = sym_op.transformHKL([0, 1, 0])
+            S[:, 2] = sym_op.transformHKL([0, 0, 1])
+            if _hkl:
+                UBSW = np.linalg.multi_dot([UB, S, W])
+            else:
+                UBSW = S
```

Comment:
Small change. This should be `UBSW = np.dot([S, W])`. See the line in MDNorm https://github.com/mantidproject/mantid/blob/3c3d606b7669241044f63d5130c97fda5efea10c/Framework/MDAlgorithms/src/MDNorm.cpp#L1103